bin/sparql-to-json

```diff
@@ -16,5 +16,20 @@ source.setEncoding('utf8');
 source.on('data', function (data) { query += data; });
 source.on('end', function () {
   var parseTree = new SparqlParser().parse(query);
-  process.stdout.write(JSON.stringify(parseTree, null, '  ') + '\n');
+  function termReplacer(key, value) {
+    if (isQuad(value)) {
+      // Workaround for N3 Quad.toJSON() missing `termType` property
```

Comment:
@simonvbrae Do we need to fix this in N3?

Reply:
Hmm I thought so too initially. I checked to see if I could find the problem in N3 but I didn't see a problem there... @AlexeyMz Why do we need this workaround?